src/util/android/event_loop_signal.hpp

```diff
@@ -134,6 +167,37 @@ class EventLoopSignal : public std::enable_shared_from_this<EventLoopSignal<Call
         // return 1 to continue receiving events
         return 1;
     }
+
+    // Write a byte to a pipe to notify anyone waiting for data on the pipe
+    static void notify_fd(int fd)
```

Comment:
Are the multiple places some of the different event loop signal implementations across platforms? It might be a little tricky to find a shared home as not every implementation wants it.